app/src/main/res/layout/activity_main.xml

```diff
@@ -9,7 +9,8 @@
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recyclerview_allsightings"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        />

```

Comment:
We've been keeping the /> on the same line as the last "thing" in all the other files